module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,13 @@
+#include "sort.hpp"
+
+#include <set>
+
+std::deque<std::string> lengthSort(const std::forward_list<std::string>& words) {
+    std::multiset<std::string, decltype(compare)> ms(compare);
```

Comment:
ms is not good name here.

Reply:
Indeed, but 'multiset' removed anyway in favour of 'sort' with lambda (as suggested below).